index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>JSON Tester</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body onload="renderHello()">
```

Comment:
you might alreay know this, but this can also be done without the `onload` if you leave your `script` tags at the bottom of your body. this way you dont need an implicit dependency on your script tag